Holo-annie-jam/Managers/RhythmManager/BeatmapPlayer.cs

```diff
@@ -13,7 +13,7 @@ public class BeatmapPlayer {
     /// <summary>
     /// The default timespan that is visible on the screen when playing a beatmap, in ticks (100 nanoseconds).
     /// </summary>
-    public const long DEFAULT_VISIBLE_TIMESPAN_TICKS = 2 * TimeSpan.TicksPerSecond;
+    public const long DEFAULT_VISIBLE_TIMESPAN_TICKS = (long)(1.1 * TimeSpan.TicksPerSecond);
```

Comment:
I prefer 1100 * TimeSpan.TicksPerMillisecond

Reply:
you right, i'm just lazy and want to use const. Is it ok?

Reply:
oh wait nvm I got u